code/datums/map_config.dm

```diff
@@ -140,4 +140,4 @@
 		. += "_maps/[map_path]/[file]"
 
 /datum/map_config/proc/MakeNextMap()
-	return config_filename == "data/next_map.json" || fcopy(config_filename, "data/next_map.json")
+	return config_filename == "data/next_map.json" || fcopy(config_filename, "data/next_map.json")
```

Comment:
```suggestion
	return config_filename == "data/next_map.json" || fcopy(config_filename, "data/next_map.json")

```

Reply:
pfft my text editor and not just a terrible habit i have